pihole

```diff
@@ -375,8 +375,10 @@ Add '-h' after specific commands for more information on usage
 Whitelist/Blacklist Options:
   -w, whitelist       Whitelist domain(s)
   -b, blacklist       Blacklist domain(s)
-  --wild, wildcard     Wildcard blacklist domain(s)
   --regex, regex       Regex blacklist domains(s)
+  --whiteregex         Regex whitelist domains(s)
```

Comment:
This should have a hyphen between `white` and `regex`. Same below with `white` and `wild`.